### PR TITLE
Issue #15456: Specify violation messages for SealedShouldHavePermitsListCheck

### PR DESCRIPTION
**Pull request by Mostafahassen1**
Issue #15456: Specify violation messages for SealedShouldHavePermitsListCheck

### CODE REVIEW

File: src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/design/sealedshouldhavepermitslist/InputSealedShouldHavePermitsListInnerClass.java [hunk 1]
```diff
@@ -7,12 +7,13 @@
 package com.puppycrawl.tools.checkstyle.checks.design.sealedshouldhavepermitslist;
 
 public class InputSealedShouldHavePermitsListInnerClass {
-    sealed class A {} // violation
+    sealed class A {} // violation 'Sealed classes or interfaces should explicitly declare'
```
Comment (romani) [inline src/test/resources-noncompilable/com/puppycrawl/tools/checkstyle/checks/design/sealedshouldhavepermitslist/InputSealedShouldHavePermitsListInnerClass.java:10]: Please use last part of message to keep it more clear`'should explicitly declare permitted subclasses'`

**Review by romani — CHANGES_REQUESTED** [review]
Item to improve:

**Review by Mostafahassen1 — COMMENTED** [review]
I finished from update

**Review by romani — APPROVED** [review]
Thanks a lot, please do more contributions

**Pull request merged**